src/App.js

```diff
@@ -1,27 +1,74 @@
 import React, { Component } from 'react';
 import './App.css';
 import GroupList from './GroupList';
+import TodoList from './TodoList';
 
 class App extends Component {
   state = {
-    groupList: [{ name: 'sally' }, { name: 'sun' }],
+    groupList: [
+      {
+        name: 'coding',
+        todoList: [
+          { content: 'todo app', completed: false },
+          { content: 'next.js', completed: false },
+        ],
+      },
+    ],
     selectedIndex: null,
   };
 
   addGroup = (name) => {
+    const { groupList } = this.state;
+
+    this.setState({
+      groupList: [...groupList, { name, todoList: [] }],
+      selectedIndex: groupList.length,
+    });
```

Comment:
```suggestion
    this.setState(({ groupList }) => {
      groupList: [...groupList, { name, todoList: [] }],
      selectedIndex: groupList.length,
    });
```
현재 state를 기반으로 업데이트를 할 때는 콜백함수를 전달하는 것이 어떨까요?

Reply:
가독성이 더 좋고 간결해졌네요!